lms/djangoapps/ccx/views.py

```diff
@@ -477,7 +477,8 @@ def prep_course_for_grading(course, request):
     field_data_cache = FieldDataCache.cache_for_descriptor_descendents(
         course.id, request.user, course, depth=2)
     course = get_module_for_descriptor(
-        request.user, request, course, field_data_cache, course.id)
+        request.user, request, course, field_data_cache, course.id, course=course
+    )
```

Comment:
Passing `course.id` and `course` opens the door to them being out-of-sync. Is there at least an assertion that they're equivalent in the code beneath that needs both? (I'll check also.)